cmd/image/qcow2ova/prep/templates.go

```diff
@@ -135,6 +135,28 @@ cloud_final_modules:
  - final-message
  - power-state-change
  - reset_rmc
+
+ ### Explicit steps for growing partitions
```

Comment:
How about adding a comment of growpart failure with multipath disks and may be a reference to public bugzilla ?